wagtail/permission_policies/pages.py

```diff
@@ -166,7 +166,7 @@ def instances_with_direct_explore_permission(self, user):
             return Page.objects.filter(depth=1)
         else:
             codenames = self._get_permission_codenames(
-                {"add", "change", "publish", "lock"}
+                {"add", "change", "publish", "lock", "unlock"}
```

Comment:
As mentioned in https://github.com/wagtail/wagtail/issues/12628#issuecomment-2559800970, while we're here, we should also add the `bulk_delete` permission here to be consistent with other "user has any page permission" checks